web/source/kmwosk.js

```diff
@@ -660,19 +700,40 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
         case 'K_SHIFT':
           nextLayer = 'shift'; break;
         case 'K_LCONTROL':
-        case 'K_RCONTROL':
         case 'K_LCTRL':
+          if(keymanweb.isChiral()) {
+            nextLayer = 'leftctrl';
+            break;
+          }
+        case 'K_RCONTROL':
         case 'K_RCTRL':
+          if(keymanweb.isChiral()) {
```

Comment:
`K_LCONTROL`/`K_LCTRL` will drop through to this and give wrong `nextLayer`

Reply:
It will only drop through if `keymanweb.isChiral() == false` - note the break in the conditionals above.  The fall-through is deliberate for the `isChiral() == false` case.

Reply:
Okay, missed that. Thanks.